autoware.repos

```diff
@@ -0,0 +1,53 @@
+repositories:
+  autoware/autoware:
+    type: git
+    url: git@github.com:autowarefoundation/autoware.git
```

Comment:
Will this repo end up being empty? If we follow the ROS2 structure, the ros2 repository only contains a repos file and a README.

Reply:
That's the goal, although something that needs to be discussed is whether project-wide documentation should be in this repo or its own. ROS 2 uses a dedicated repo, and currently that's what I've set up.